drivers/sensor/mhz19b/mhz19b.c

```diff
@@ -0,0 +1,324 @@
+/*
+ * Copyright (c) 2021 G-Technologies Sdn. Bhd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "mhz19b-priv.h"
+
+#if !IS_ENABLED(CONFIG_UART_INTERRUPT_DRIVEN)
```

Comment:
You have a dependency in Kconfig on `UART_INTERRUPT_DRIVEN` so it seems that it's not needed.

Reply:
True, fixed